conf/application.conf

```diff
@@ -60,7 +60,7 @@ logger.play=INFO
 # Logger provided to your application:
 logger.application=DEBUG
 
-kafka-manager.zkhosts="kafka-manager-zookeeper:2181"
+kafka-manager.zkhosts="localhost:2181"
```

Comment:
Let's keep this as kafka-manager-zookeeper.  This way you can always point this host to whatever host you want via /etc/hosts for example for testing.

```
#example hosts file
127.0.0.1 localhost kafka-manager-zookeeper
```